src/Bookings.js

```diff
@@ -1,18 +1,44 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResult.js";
+import ShowProfile from "./CustomerProfile.js";
+import FakeBookings from "./data/fakeBookings.json";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+
+  useEffect(() => {
+    console.log("page loaded");
+    fetch("https://cyf-react.glitch.me").then(res =>
+      res.json().then(data => {
+        console.log(data);
+        setBookings(data);
+      })
+    );
+  }, []);
+
   const search = searchVal => {
-    console.info("TO DO!", searchVal);
+    if (searchVal) {
+      const searchResult = bookings.filter(
+        singleBooking =>
+          singleBooking.firstName.toLowerCase() === searchVal.toLowerCase() ||
+          singleBooking.surname.toLowerCase() === searchVal.toLowerCase()
+      );
+      setBookings(searchResult);
```

Comment:
This is overwriting your bookings list that you got from the API. So when you filter this way, you can never get your previous results back. Have a read of this to see another way of doing things: https://www.freecodecamp.org/news/search-and-filter-component-in-reactjs/